### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by chetak123**
Signed-off-by: Ayushman Mishra <ayushvidushi01@gmail.com>

Issue : #14

### CODE REVIEW

**Comment by PushkarJ** [comment]
/cc @savitharaghunathan 

(since the relevant issue #14 is a duplicate of #20, that you created)

File: README.md [hunk 1]
```diff
@@ -1,29 +1,67 @@
...
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Security Special Interest Group
 
-## Community, discussion, contribution, and support
+Covers horizontal security initiatives for the Kubernetes project, including regular security audits, the vulnerability management process, cross-cutting security documentation, and security community management.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
```
Comment (savitharaghunathan) [inline README.md:17]: Link to charter doesn't exist in sig-security repo. Please update this link to https://github.com/kubernetes/community/blob/master/sig-security/charter.md

File: README.md [hunk 2]
```diff
@@ -1,29 +1,67 @@
...
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+* Ian Coldwater (**[@IanColdwater](https://github.com/IanColdwater)**), Twilio
+* Tabitha Sable (**[@tabbysable](https://github.com/tabbysable)**), Datadog
 
-[owners]: https://git.k8s.io/community/contributors/guide/owners.md
-[Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
+## Contact
+- Slack: [#sig-security](https://kubernetes.slack.com/messages/sig-security)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-security)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fsecurity)
```
Comment (savitharaghunathan) [inline README.md:32]: Add an entry for sig-security issues - https://github.com/kubernetes/sig-security/issues

**Review by savitharaghunathan — COMMENTED** [review]
Couple of nits. Otherwise lgtm.

**Comment by chetak123** [comment]
Hey @savitharaghunathan can you please review my PR and give me some helpful suggestions if required.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,68 @@
...
+- [Sig-Security Issues](https://github.com/kubernetes/sig-security/issues)
```
Comment (PushkarJ) [inline README.md:33]:
```suggestion
- [SIG Security Issues](https://github.com/kubernetes/sig-security/issues)
```

**Comment by raesene** [comment]
Hey all , I was just thinking "it'd be nice if we had a README" then realise this was here :) what do we think needs done to get this one merged?

File: README.md [hunk 4]
```diff
@@ -1,29 +1,68 @@
...
-- an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
```
Comment (mtardy) [inline README.md:8]:
```suggestion

```

File: README.md [hunk 5]
```diff
@@ -1,29 +1,68 @@
...
+SIG Security discussions, documents, processes and other artifacts
+- **Owners:**
+  - [kubernetes/sig-security](https://github.com/kubernetes/sig-security/blob/master/OWNERS)
+- **Contact:**
+  - Slack: [#sig-security](https://kubernetes.slack.com/messages/sig-security)
+
+[subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
+<!-- BEGIN CUSTOM CONTENT -->
+
+<!-- END CUSTOM CONTENT -->
```
Comment (mtardy) [inline README.md:68]:
```suggestion

```

**Review by mtardy — COMMENTED** [review]
I tried all the URLs and everything seems correct!

Just a nit to remove the template comments in the markdown that are no longer needed but do not appear anyway so...

Sorry for not reviewing sooner and thanks so much for taking the time to write the SIG readme!

File: README.md [hunk 6]
```diff
@@ -1,29 +1,68 @@
...
+- Slack: [#sig-security](https://kubernetes.slack.com/messages/sig-security)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-security)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fsecurity)
+- [Sig-Security Issues](https://github.com/kubernetes/sig-security/issues)
+- GitHub Teams:
+    - [@kubernetes/sig-security-leads](https://github.com/orgs/kubernetes/teams/sig-security-leads) - SIG Security Leads
+    - [@kubernetes/sig-security-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-security-pr-reviews) - SIG Security PR review notifications
+- Steering Committee Liaison: Paris Pittman (**[@parispittman](https://github.com/parispittman)**)
+
+## Subprojects
```
Comment (savitharaghunathan) [inline README.md:39]: Add info about assessments sub-project. 
link to owners file: https://github.com/kubernetes/sig-security/blob/main/sig-security-assessments/OWNERS

**Review by savitharaghunathan — COMMENTED** [review]
one minor suggestion. Otherwise, lgtm

**Comment by PushkarJ** [comment]
Thanks for working on those changes @chetak123 and everybody else for the feedback!

/lgtm
/assign @tabbysable  @IanColdwater

**Comment by IanColdwater** [comment]
Hi Ayushman! Thank you so much for contributing this README! We really appreciate your work. :)

We as chairs have been wanting to add language that accurately reflects who we are and what we do as a SIG, because SIG Security has evolved a lot since the original charter was written. We are currently workshopping that language and plan to suggest that to you as an inline change next week.

Thank you so much for your patience, and looking forward to merging this!

**Comment by nikhita** [comment]
/remove-lifecycle stale

@tabbysable bump on this. looks like the README still says `Kubernetes Template Project` :)

**Comment by mtardy** [comment]
so much anticipation 👀 ...

**Comment by tabbysable** [comment]
For the record, and the joy of doing so:
/approve
/honk

**Comment by PushkarJ** [comment]
Thank you @chetak123 for working on this and for your patience while we all worked out what was best for our beloved SIG 🎉

**Pull request merged**